extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,39 +46,20 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state> with json child
```

Comment:
@kmh287 Yeah I saw that as well. We could remove the child_tags unless that would be bad to have two JSON scripts within an amp-state.